packages/gatsby/cache-dir/ssr-develop-static-entry.js

```diff
@@ -111,7 +115,66 @@ export default (pagePath, isClientOnlyPage, callback) => {
 
     const pageData = getPageData(pagePath)
 
-    const componentChunkName = pageData?.componentChunkName
+    const { componentChunkName, staticQueryHashes = [] } = pageData
+
+    let scriptsAndStyles = flatten(
+      [`commons`].map(chunkKey => {
+        const fetchKey = `assetsByChunkName[${chunkKey}]`
+
+        let chunks = get(stats, fetchKey)
+        const namedChunkGroups = get(stats, `namedChunkGroups`)
```

Comment:
I copy/pasted this from static-entry.js which was written a long long time ago pre fancy-dancy stuff like `?`. Is there a reason to refactor other than preference? I'd rather keep the implementation of the two the same.

Reply:
The main reason sis technical debt. I don't think this is a preference but rather about redundant outdated code.

My suggestion, if there's a strong desire to keep them, is to share this code somehow, rather than to dedupe it. It may be copy/paste now, but in some future somebody will look at this code and not realize that this is the case, just like I didn't know when I was reviewing this.

If nothing else, and there's a desire to keep them in sync, there ought to be a comment or notice about this somewhere in the file.

Reply:
extracting out to a common function is a great idea — I hadn't done that earlier as things were in flux but let's do that & then we can refactor/modernize the code

Reply:
Let's do this separately, extracting to common util can have some impact on regular ssr, and we want to backport this for tomorrow's release so it's safer to ship this as-is (as everything here touches pretty much things behind the flag) and we can make common util (and also address outdated code / unneeded use of lodash too)